<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookRoomDialog</class>
 <widget class="QDialog" name="BookRoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="lblAddress">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLineEdit" name="txtContactNo"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="lblIdProof">
       <property name="text">
        <string>Id Proof</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="lblContactNo">
       <property name="text">
        <string>Contact No</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QLineEdit" name="txtIdProof"/>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QPlainTextEdit" name="plainTextEditAddress"/>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLineEdit" name="txtName"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="lblSelectRoom">
       <property name="text">
        <string>Select Room:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QComboBox" name="cmbBoxRooms"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnSubmit">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
